build/detect-upstream-tests/mock_makefile_pass.txt

```diff
@@ -0,0 +1,340 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Copyright Contributors to the Open Cluster Management project
+
+PWD := $(shell pwd)
+LOCAL_BIN ?= $(PWD)/bin
+
+# Keep an existing GOPATH, make a private one if it is undefined
+GOPATH_DEFAULT := $(PWD)/.go
+export GOPATH ?= $(GOPATH_DEFAULT)
+GOBIN_DEFAULT := $(GOPATH)/bin
+export GOBIN ?= $(GOBIN_DEFAULT)
+export PATH := $(LOCAL_BIN):$(GOBIN):$(PATH)
+GOARCH = $(shell go env GOARCH)
+GOOS = $(shell go env GOOS)
+TESTARGS_DEFAULT := -v
+export TESTARGS ?= $(TESTARGS_DEFAULT)
+VERSION ?= $(shell cat COMPONENT_VERSION 2> /dev/null)
+IMAGE_NAME_AND_VERSION ?= $(REGISTRY)/$(IMG)
+# Handle KinD configuration
+KIND_NAME ?= test-managed
+KIND_NAMESPACE ?= open-cluster-management-agent-addon
+KIND_VERSION ?= latest
+MANAGED_CLUSTER_NAME ?= managed
+WATCH_NAMESPACE ?= $(MANAGED_CLUSTER_NAME)
+ifneq ($(KIND_VERSION), latest)
+	KIND_ARGS = --image kindest/node:$(KIND_VERSION)
+else
+	KIND_ARGS =
+endif
+# Test coverage threshold
+export COVERAGE_MIN ?= 75
+
+# Image URL to use all building/pushing image targets;
+# Use your own docker registry and image name for dev/test by overridding the IMG and REGISTRY environment variable.
+IMG ?= $(shell cat COMPONENT_NAME 2> /dev/null)
+REGISTRY ?= quay.io/stolostron
```

Comment:
We should make sure references to Quay match up, also. It'll be here and in the manifests.

Reply:
I suppose I'm assuming that's in the scope of this script?

Reply:
That's a good point @dhaiducek.